packages/main/src/Input.js

```diff
@@ -11,7 +11,8 @@ import {
 } from "@ui5/webcomponents-base/dist/Keys.js";
 import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
 import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
-// import Icon from "./Icon.js";
+import "@ui5/webcomponents-icons/dist/icons/decline.js";
```

Comment:
I think the Input itself does not depend on those icons, the icon is a slot, and should be imported by the one that uses the ui5-input, but maybe I missed where those icons are used.

Reply:
You're right, reverted the change to Input.js